sdk/src/main/java/com/ingresse/sdk/services/UserService.kt

```diff
@@ -205,6 +211,63 @@ class UserService(private val client: IngresseClient) {
         mUpdateBasicInfosCall?.enqueue(RetrofitCallback(type, callback))
     }
 
+    /**
+     * Update user picture
+     *
+     * @param request - all parameters used for update user picture
+     * @param onSuccess - success callback
+     * @param onError - error callback
+     * @param onTokenExpired - user token expired callback
+     */
+    fun updateUserPicture(request: UserPicture,
+                         onSuccess: (UserUpdatedDataJSON) -> Unit,
+                         onError: ErrorBlock,
+                         onNetworkError: Block,
+                         onTokenExpired: Block) {
+
+        request.picture = "data:image/png;base64,${request.picture}"
+        mUpdateUserPictureCall = service.updateUserPicture(
+            userId = request.userId,
+            apikey = client.key,
+            userToken = request.userToken,
+            params = request
+        )
+
+        val callback = object : IngresseCallback<Response<UserUpdatedJSON>?> {
+            override fun onSuccess(data: Response<UserUpdatedJSON>?) {
+                val response = data?.responseData ?: return onError(APIError.default)
+
+                if (!response.message.isNullOrEmpty()) {
+                    val apiError = APIError()
+                    apiError.message = response.message.joinToString(", ")
+                    apiError.title = "Verifique suas informações"
+                    apiError.code = 0
+                    onError(apiError)
+                    return
+                }
+
+                if (response.status == null) return onError(APIError.default)
+
+                if (response.status) {
+                    response.data?.let { obj -> onSuccess(obj) }
+                    return
+                }
+
+                val responseData = response.data ?: return onError(APIError.default)
```

Comment:
You could use this for the last if and avoid unnecessary let or empty callback if's null.